README.md

```diff
@@ -19,10 +19,11 @@ please note that if the cluster is stoped or doesn't exits the application will
 
 1. Clone this repository in your local. `git clone https://github.com/Felipeg005/FrameNoob-Blog`
 2. Checkout the correct branch `git checkout <desired branch>`
-3. Create the database of the project and load default data in db/seed.rb with this command
+3. Please run `install bundle` to install all necessary gems before set up the database
```

Comment:
- [ ] I think this is not the right command to install the gem. It should be `bundle install` to install the necessary gem.

Reply:
Yes totally right, now it's fixed 👍